brayns/common/scene/Model.h

```diff
@@ -189,6 +190,27 @@ class Model
      * now on.
      */
     BRAYNS_API void addModel(ModelPtr model, const Transformations& transform);
+    
+    /**
+      Adds a SDFGeometry to the scene
+      @param materialId Material of the geometry
+      @param geom Geometry to add
+      @param neighbours List of global indices of the geometry neighbours
```

Comment:
For people like me who are not familiar with SDF terminology, it would great if you could document what neighbours are used for and how I should compute them.